### PR TITLE
Feat display base attributes

### PR DESCRIPTION
**Pull request by Darxo**
### Brother

image

image

image

image

image

image

image

image

### Dummyplayer

image

image

### CODE REVIEW

File: mod_reforged/hooks/config/character.nut
Comment (LordMidas) [inline mod_reforged/hooks/config/character.nut:6]: This seems like a useful function and should be added via MSU imo. For now we can keep it in Reforged but call it `getFatigueRecoveryRate`

File: mod_reforged/hooks/items/item_container.nut
Comment (LordMidas) [inline mod_reforged/hooks/items/item_container.nut:3]: This can be reduced to `return -this.getStaminaModifier(_slots)` because that function should _always_ return a negative number.

For now I think addition of this function is not good, we should wait until we have a proper weight system. This is only really used in one place right now and that place can instead use `-getStaminaModifier(_slots)` instead.

File: mod_reforged/hooks/ui/screens/tooltip/tooltip_events.nut
Comment (LordMidas) [inline mod_reforged/hooks/ui/screens/tooltip/tooltip_events.nut:1]: This function is only used down below inside another function _once_. Therefore, having it as a local function doesn't really benefit anything except perhaps readability of the other function. Because it is local, no submodder can hook it and change anything. I'd suggest making it a proper new function inside that class so that modders could tweak it if they want.

File: mod_reforged/hooks/ui/screens/tooltip/tooltip_events.nut
Comment (LordMidas) [inline mod_reforged/hooks/ui/screens/tooltip/tooltip_events.nut:113]: I'd change this to `Items Weight:` and color it red.

File: mod_reforged/hooks/ui/screens/tooltip/tooltip_events.nut
Comment (LordMidas) [inline mod_reforged/hooks/ui/screens/tooltip/tooltip_events.nut:3]: Remove parantheses before ternary

File: mod_reforged/hooks/ui/screens/tooltip/tooltip_events.nut
Comment (LordMidas) [inline mod_reforged/hooks/ui/screens/tooltip/tooltip_events.nut:1]: A better name could be `getBaseAttributesTooltip`

File: mod_reforged/config/config.nut
Comment (LordMidas) [inline mod_reforged/config/config.nut:2]: The changes about the hiring cost variance have snuck into this PR and should be removed.

**Review by LordMidas — CHANGES_REQUESTED** [review]

**Review by LordMidas — APPROVED** [review]
Looks good to me. I pushed a commit to fix a typo in a comment. I do not know if the js hack in this PR is the best approach, I'd suggest asking @TaroEld or @Enduriel for confirmation and then merge.

**Comment by Darxo** [comment]
@LordMidas Are we still waiting for the tech teams approval considering they seem to be busy these days?

**Comment by LordMidas** [comment]
> @LordMidas Are we still waiting for the tech teams approval considering they seem to be busy these days?

Endy is busy but I think Taro should be able to take a quick look. I'll request him.

**Comment by LordMidas** [comment]
Taro said it looks good. So it's good to merge now.

**Pull request merged**
